client/src/components/ElementEditor/InlineEditForm.js

```diff
@@ -1,11 +1,45 @@
+/* global window */
 import React, { PureComponent, PropTypes } from 'react';
 import classnames from 'classnames';
 import FormBuilderLoader from 'containers/FormBuilderLoader/FormBuilderLoader';
 import { loadElementSchemaValue } from 'state/editor/loadElementSchemaValue';
+import i18n from 'i18n';
 
 class InlineEditForm extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.handleLoadingError = this.handleLoadingError.bind(this);
+
+    this.state = {
+      loadingError: null
+    };
+  }
+
+  handleLoadingError() {
+    const { jQuery: $ } = window;
+    const { handleLoadingError } = this.props;
+
+    this.setState({
+      loadingError: true,
+    });
+
+    $.noticeAdd({
+      text: i18n.inject(
+        i18n._t(
+          'EditForm.ERROR_NOTIFICATION',
+          'Error rendering the edit form for this block'),
+        ),
+        stay: false,
```

Comment:
I'm still not sold on using a toast message for this. It will disappear quickly and people might miss it, look back at their screen and not know why something is broken. Can you convince me otherwise? =D